articles/libraries/lock/v10/lock-passwordless-migration-guide.md

```diff
@@ -0,0 +1,162 @@
+
+## General Changes and Additions
+
+### Importing Auth0LockPasswordless
+
+If you're loading from the CDN, you can still use `Auth0LockPasswordless`. The difference is that you'll provide options in the constructor, like we do with `Auth0Lock`:
+
+#### Using the CDN
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li><a href="#cdn-before" data-toggle="tab">Before</a></li>
+      <li><a href="#cdn-after" data-toggle="tab">After</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="cdn-before" class="tab-pane active">
+      <pre class="hljs html"><code>
+        &lt;script src=&quot;http://cdn.auth0.com/js/lock-passwordless-2.2.3.min.js&quot;&gt;&lt;/script&gt;
+        &lt;script&gt;
+          var lock = new Auth0LockPasswordless(clientID, domain);
+        &lt;/script&gt;
+      </code></pre>
+    </div>
+    <div id="cdn-after" class="tab-pane">
+      <pre class="hljs html"><code>
+        &lt;script src=&quot;https://cdn.auth0.com/js/lock/10.x.y/lock.min.js&quot;&gt;&lt;/script&gt;
+        &lt;script&gt;
+          var options = {
+            oidcConformant: true
+          }
+          var lock = new Auth0LockPasswordless(clientID, domain, options);
+        &lt;/script&gt;
+      </code></pre>
+    </div>
+  </div>
+</div>
+
+#### Using npm + module bundler
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li><a href="#npm-before" data-toggle="tab">Before</a></li>
+      <li><a href="#npm-after" data-toggle="tab">After</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="npm-before" class="tab-pane active">
+      <pre class="hljs js"><code>
+        import Auth0LockPasswordless from 'auth0-lock-passwordless';
+        var lock = new Auth0LockPasswordless(clientID, domain);
+      </code></pre>
+    </div>
+    <div id="npm-after" class="tab-pane">
+      <pre class="hljs js"><code>
+        import Auth0LockPasswordless from 'auth0-lock/passwordless';
+        var options = {
+          oidcConformant: true
+        };
+        var lock = new Auth0LockPasswordless(clientID, domain, options);
+      </code></pre>
+    </div>
+  </div>
+</div>
+
+### Initialization Options
+
+`Auth0LockPasswordless` has the same [options available](/libraries/lock/v10/customization) as `Auth0Lock` in addition to a single new option that determines if you want to use a Magic Link or a Email Code when using an email passwordless connection. For this property, `passwordlessMethod`, only two values are accepted:
+
+- `code` if you want to use an Email Code
+- `link` if you want to use a Magic Link
+
+```js
+var options = {
+  oidcConformant: true,
+  passwordlessMethod: 'code' //or link
```

Comment:
This would look better if you used the Before/After UI (with captions: Email Code/Magic Link)